dotnet-authserver/src/TeacherIdentity.AuthServer/Models/Mappings/EstablishmentDomainMapping.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace TeacherIdentity.AuthServer.Models.Mappings;
+
+public class EstablishmentDomainMapping : IEntityTypeConfiguration<EstablishmentDomain>
+{
+    public void Configure(EntityTypeBuilder<EstablishmentDomain> builder)
+    {
+        builder.ToTable("establishment_domains");
```

Comment:
Worth throwing on the case insensitive collation here?

Reply:
Yep - done